### DISCUSSION TITLE
Packagemanifest format will be deprecated

### DISCUSSION THREAD

**Discussion by J0zi** [redhat-openshift-ecosystem/community-operators-prod#2372] [Announcements]
Dear OpenShift Operator Community,

`Packagemanifest` format will no longer be supported from July 2023. The only supported format will be [`bundle`](https://docs.openshift.com/container-platform/4.12/operators/understanding/olm-packaging-format.html#olm-bundle-format_olm-packaging-format) format. You can migrate using `operator-sdk` binary, please follow the [instructions](https://docs.openshift.com/container-platform/4.12/operators/operator_sdk/osdk-pkgman-to-bundle.html).

Anyone who has not migrated by June 30th will be auto-converted to bundle format in a one-off script we are running. More information about the script later. Please stay tuned.

Kind Regards,
Community operators team

mentions:
@2uasimojo, @3scale, @ArangoGutierrez, @ArbelNathan, @BaruchBilanski, @DanArlowski, @DanielePalaia, @FlorentinDUBOIS, @FushuWang, @HammerMeetNail, @IanBoden, @J0113, @J0zi, @Jainbrt, @JaydipGabani, @JohnSnowLabs, @JohnStarich, @JohnStrunk, @Jooho, @Julien-Ben, @KanShiori, @KohlsTechnology, @KoppulaRajender, @LaVLaS, @NissesSenap, @RafalSkolasinski, @RobotSail, @SergK, @Sosivio, @Sosivio-Bot, @Sugar-pack, @SumanthKodali999, @VaishnaviHire, @ValClarkson, @VineethReddy02, @abutcher, @ack-bot, @acm-canary-bot, @acornett21, @adambkaplan, @ahadas, @akashkumar1982, @akhil-rane, @alien-ike, @alinaryan, @andreaTP, @andrewlecuyer, @ashashiv, @ashenoy-hedvig, @ashishshinde, @becash143, @beekhof, @benjaminjb, @burmanm, @camilamacedo86, @cap1984, @carlosedp, @carlosthe19916, @celenechang, @cgoodfred, @chanwit, @che-incubator, @che-incubator-bot, @cheels, @christophd, @cliveseldon, @cnbu-jenkins, @cooktheryan, @cschwede, @ctrought, @datianshi, @davi17g, @davidebianchi, @davidesalerno, @devOpsHelm, @devopstales, @didierofrivia, @dkwon17, @dlom, @dmesser, @droopy4096, @dt-team-kubernetes, @dymurray, @eguzki, @ejweber, @esara, @external-secrets, @f41gh7, @fabritsius, @fao89, @fbladilo, @frzifus, @gauravshankarcan, @ggeorgiev-gitlab, @gl-distribution-oc, @gsreg, @guicassolato, @gusfcarvalho, @haibinxie, @hasancelik, @hasheddan, @hco-bot, @helderjs, @iam-veeramalla, @iamabhishek-dubey, @iblancasa, @igor-karpukhin, @ishitasequeira, @isimluk, @james-cccc, @jbusche, @jeesmon, @jharrington22, @jjaferson, @jkatz, @jlopes-randoli, @jmazzitelli, @jmontleon, @joelddiaz, @jonathankingfc, @jonathanvila, @jotak, @khewonc, @knelasevero, @ktenzer, @kube-green, @lbroudoux, @leo-ri, @linhlam-kc, @maistra, @makon57, @mariomac, @maskarb, @matancarmeli7, @mbaldessari, @mgubaidullin, @microcks, @mitchnielsen, @mmontes11, @moolen, @morvencao, @mpryc, @mrizzi, @mshitrit, @mtyazici, @muvaf, @mvala, @mvalarh, @ndmspc, @nickboldt, @nmarukovich, @npecka, @opdev, @openshift-edge-bot, @oriyarde, @oscerd, @osherdp, @paulofrazao, @pavolloffay, @periklis, @phantomjinx, @piyush-nimbalkar, @portworx, @ppatierno, @pure-jliao, @pursultani, @qbarrand, @quay, @quay-devel, @rabbitmq, @raffaelespazzoli, @randoli, @ravienggtoo, @rawagner, @rayfordj, @razo7, @redhat-cop, @redhat-openshift-ecosystem, @rensyct, @rgolangh, @ricardozanini, @rigazilla, @rkukura, @rmr-silicom, @roivaz, @rokej, @rooftopcellist, @roypaulin, @rubenvp8510, @rurikudo, @ruromero, @rvansa, @ryanemerson, @sabre1041, @scholzj, @secondsun, @selansen, @selvamt94, @silicomDK, @skanthed, @skattoju, @snyk, @snyksec, @spabba17, @spilchen, @spolti, @stolostron, @streamnative, @streamnativebot, @sud82, @suhanime, @sunya-ch, @tadayosi, @tedhtchang, @tjmoore4, @tolusha, @turbodeploy, @turbonomic, @turkenh, @vaibhavjainwiz, @vanessarakoczy, @vboulineau, @vijtrip2, @vmuzikar, @wallrj, @xiangjingli, @yaalsn, @yossig-aquasec, @yuriolisa, @zhiweiyin318, @zregvart, @zroubalik

**Comment by jpaulovic** [comment]
Dear users, the support of packagemanifest format was officially ceased as of today. Going forward, please submit all your operators in the bundle format.
All existing operators using the packagemanifest format but one have been converted to the bundle format.

Best regards,
Community operators team
